install.js

```diff
@@ -132,7 +132,9 @@ function findSuitableTempDirectory() {
 
   for (let i = 0; i < candidateTmpDirs.length; i++) {
     if (!candidateTmpDirs[i]) continue;
-    const candidatePath = path.join(candidateTmpDirs[i], 'chromedriver');
+    // Prevent collision with other versions in the dependency tree
+    const namespace = Math.random().toString(36).substr(2);
+    const candidatePath = path.join(candidateTmpDirs[i], namespace, 'chromedriver');
```

Comment:
You are using a random, it would be better to use the version number to namespace the file, so it can be cached.

Reply:
done

Reply:
You should be using `chromedriver_version`, because the user can ask for a specific version (see [versioning](https://github.com/giggio/node-chromedriver#versioning)). The only problem is if the user supplies `LATEST`. Then we have a problem because `tmpPath` is set before we check for the latest version.
I suggest you wait for the version check here:
https://github.com/giggio/node-chromedriver/blob/3cb3d6abc1c9c49c0d9f9058026b421f081861c2/install.js#L52-L55
Add code a call to `findSuitableTempDirectory` right after that and before checking if we have a cache:
https://github.com/giggio/node-chromedriver/blob/3cb3d6abc1c9c49c0d9f9058026b421f081861c2/install.js#L56
I checked the data flow and I think it should be doable. We don't use `tmpPath`, `chromedriverBinaryFileName`, `chromedriverBinaryFilePath` or `downloadedFile` before that.

Reply:
I made that change.